lib/gems/pending/VMwareWebService/MiqVimInventory.rb

```diff
@@ -64,11 +65,17 @@ def initialize(server, username, password, cacheScope = nil)
         deleteProperty(:HostSystem, "config.dateTimeInfo")
       end
     else
-      if @v4 && cacheScope != :cache_scope_event_monitor
+      if @v4
```

Comment:
Is the removal of the other conditions related to this PR? Is there no longer a need to check the cache scope?